lib/openhab/core/dsl/items/datetime_item.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require 'forwardable'
+require 'java'
+require 'time'
+require 'openhab/core/dsl/types/datetime'
+
+module OpenHAB
+  module Core
+    module DSL
+      module Items
+        #
+        # Delegation to OpenHAB DateTime Item
+        #
+        class DateTimeItem
+          extend Forwardable
+          include Comparable
+
+          def_delegator :@datetime_item, :to_s
```

Comment:
I'd like to suggest making `state` available, i.e.:

```suggestion
          def_delegators :@datetime_item, :to_s, :state
```

It will be useful when doing comparisons from other objects. I had to make this change on NumberItem and StringItem in #135

@boc-tothefuture WDYT?

Reply:
`state` is already delegated to the item via `method_missing`, the reason `to_s` need to be explicitly delegated is because all ruby objects have that method (since they inherit from Object), so `method_missing` never gets called, and we need to make sure it's the Item's `toString()` method that gets called instead (jruby handles the name translation).

Reply:
> `state` is already delegated to the item via `method_missing`

You're correct. I re-tested my own changes without delegating `state` and they still work. I've updated my own PR. Thanks! Feel free to mark this as resolved.